pom.xml

```diff
@@ -5260,6 +5240,36 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>pl.project13.maven</groupId>
+        <artifactId>git-commit-id-plugin</artifactId>
+        <version>2.2.6</version>
+        <configuration>
+          <generateGitPropertiesFile>true</generateGitPropertiesFile>
+          <generateGitPropertiesFilename>${project.build.outputDirectory}/src/main/resources/META-INF/git.properties</generateGitPropertiesFilename>
+          <failOnNoGitDirectory>false</failOnNoGitDirectory>
+          <commitIdGenerationMode>full</commitIdGenerationMode>
+          <gitDescribe>
+            <tags>true</tags>
+            <forceLongFormat>true</forceLongFormat>
+          </gitDescribe>
+          <includeOnlyProperties>
+            <includeOnlyProperty>^git.branch$</includeOnlyProperty>
+            <includeOnlyProperty>^git.build.version$</includeOnlyProperty>
+            <includeOnlyProperty>^git.commit.id.full$</includeOnlyProperty>
+            <includeOnlyProperty>^git.tags=</includeOnlyProperty>
+          </includeOnlyProperties>
+        </configuration>
+        <executions>
+          <execution>
+            <id>get-the-git-infos</id>
+            <goals>
+              <goal>revision</goal>
+            </goals>
+            <phase>initialize</phase>
+          </execution>
+        </executions>
+      </plugin>
```

Comment:
Seems fine.
A good practice is to separate the plugin execution (`build/plugins`) from the plugin definition and configuration (`build/pluginManagement/plugins`).
The first one is short and only specify the plugin name and, if necessary, the wanted executions or the customized configuration (for the current POM only).
The second one (`pluginManagement`) is much more verbose: specifying the version, common configuration, default executions... By default, it will be inherited by the children projects.

Please, for the review, attach an output of the generated files in the ticket for a build starting from root and including one or two sub-modules.